src/project-2.js

```diff
@@ -44,63 +56,69 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+  if (num < 2) return false;
+
+  for (let i = 2; i < num; i++) {
```

Comment:
Great.  You can also have your loop skip even numbers.  And if you want to get very advanced you only need to iterate up to the square root of the number.  Check out the sieve of Eratosthenes on Wikipedia